extensions/couponpdf/src/main/java/com/liferay/couponpdf/service/config/HttpSecurityConfig.java

```diff
@@ -10,5 +10,6 @@ public class HttpSecurityConfig extends WebSecurityConfigurerAdapter {
   @Override
   protected void configure(HttpSecurity httpSecurity) throws Exception {
     httpSecurity.csrf().disable().authorizeRequests().antMatchers("/").permitAll();
+    httpSecurity.cors();
   }
 }
```

Comment:
we should have a cors pattern in the original springboot app... we need to dig it up. But we also have new properties to load the correct domains from too... we'll tune this.

Reply:
Right, I also had to modify the domains to `*` which is wrong.  Maybe you can help me investigate.